src/test/java/athena/task/taskfilter/ForecastFilterTest.java

```diff
@@ -24,37 +30,39 @@ void testIsTaskIncluded_all_returnsTrue() {
     }
 
     /**
-     * Checks if task is included after applying the week forecast filter.
+     * Checks if task in this week is included after applying the week forecast filter.
      */
     @Test
-    void testIsTaskIncluded_week_returnsTrue() {
+    void testIsTaskIncluded_filterByWeek_returnsTrue() {
+        String todayDateString = LocalDate.now().toString();
         ForecastFilter forecastFilter = new ForecastFilter(Forecast.WEEK);
         Task inputTask = new Task("testName", "0900", "1", "05-11-2020",
-                "16-10-2020", Importance.LOW, "testNotes", 0); // Tested on 12-10-2020
+                todayDateString, Importance.LOW, "testNotes", 0);
         boolean isTaskIncluded = forecastFilter.isTaskIncluded(inputTask);
         assertEquals(isTaskIncluded, true);
     }
 
     /**
-     * Checks if task is not included after applying the week forecast filter.
+     * Checks if task not in this week is not included after applying the week forecast filter.
      */
     @Test
-    void testIsTaskIncluded_week_returnsFalse() {
+    void testIsTaskIncluded_filterByWeek_returnsFalse() {
         ForecastFilter forecastFilter = new ForecastFilter(Forecast.WEEK);
         Task inputTask = new Task("testName", "0900", "1", "05-11-2020",
-                "23-10-2020", Importance.LOW, "testNotes", 0); // Tested on 12-10-2020
+                "23-10-2020", Importance.LOW, "testNotes", 0);
```

Comment:
This test will return true once 23-10-2020 is in the same week as the current date. Perhaps there's a better way to generate a false case. Maybe the recurrence can be `(LocalDate.now() + 7 days).toString()`

Reply:
Oh right good catch